frontend/src/components/posts/Comments.jsx

```diff
@@ -0,0 +1,34 @@
+//react core
+import React, { useState, useRef } from 'react';
+
+//styles
+import '../../CSS/styles.css';
+
+export default function Post({ onSubmit }) {
```

Comment:
This is supposed to be `function Comment()`, isn't it? Other than that, looks fine.

Reply:
The file is called Comments.jsx. The function is called Post...
And you return article with className footer...
very confusing

Reply:
yes its a typo, which I will correct now